test/plugins/features/movies/controller.test.js

```diff
@@ -2,6 +2,22 @@
 
 const Controller = require('../../../../lib/plugins/features/movies/controller');
 
+const Knex = require('../../../../lib/libraries/knex');
+
+const Movie = require('../../../../lib/models/movie');
+
+beforeEach('Setup test DB', async () => {
+  await Knex.raw('TRUNCATE TABLE movies CASCADE');
+  await new Movie().save({
+    title: 'Twisted',
+    release_year: 2004
+  });
+  await new Movie().save({
+    title: 'Never Die Twice',
+    release_year: 2001
+  });
```

Comment:
I think you may be able to insert multiple Movies at once using `knex` and factories! Not sure if you've set that up in previous PRs?

Reply:
Ooooo.. interesting!

Reply:
I remember this now! The next commit should use that approach. :)